README.md

```diff
@@ -285,13 +285,45 @@ import kagglehub
 # Download the latest version.
 kagglehub.competition_download('digit-recognizer')
 
-# Download a single file
+# Download a single file.
 kagglehub.competition_download('digit-recognizer', path='train.csv')
 
 # Download a competition or file, even if previously downloaded to cache. 
 kagglehub.competition_download('digit-recognizer', force_download=True)
 ```
 
+### Download Notebook Outputs
+
+The following examples download the `Titanic Tutorial` notebook output: https://www.kaggle.com/code/alexisbcook/titanic-tutorial
+
+```python
+import kagglehub
+
+# Download the latest version.
+kagglehub.notebook_output_download('alexisbcook/titanic-tutorial')
+
+# Download a specific version of the notebook output.
+kagglehub.notebook_output_download('alexisbcook/titanic-tutorial/versions/1')
+
+# Download a single file.
+kagglehub.notebok_output_download('alexisbcook/titanic-tutorial', path='submission.csv')
+```
+
+### Install Utility Script
+
+The following example installs the utility script `Physionet Challenge Utility Script` Utility Script: https://www.kaggle.com/code/bjoernjostein/physionet-challenge-utility-script
```

Comment:
nit: Not sure how far we want to go here, but you could mention like "which makes its code available to your python environment" or something.  Or just put the onus on the curious reader to dive more deeply into "hmm what does install a utility script mean?", e.g. by just going to the source for `utility_script_install`.  To that end, maybe we should have links to the source code definitions for all these core functions..? 🤔

Reply:
I think a link saying "Click here to see the `definition` (or whatever word) of this function" would be useful for the more curious user

Reply:
I think having a link after the code block works well rather than at the top. Works like a "here's a link to more if you want to look" link
https://screenshot.googleplex.com/9xy7MF9mavuuY6o